src/shared/components/flexbox/FlexBox.tsx

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
This is a file that contains the VBox and HBox components.  If you can think of a better name for the file, let me know. These VBox and HBox behave similar to their counterparts in Apache Flex.